frontend/modules/recipe/components/Ratings.js

```diff
@@ -4,19 +4,19 @@ require("./../css/ratings.scss");
 
 export default React.createClass({
   render: function() {
-    var tmp = '......';
-    var start_map = tmp.split(".", this.props.stars);
-    var empty_star_map = tmp.split(".", 5 - this.props.stars);
-    var stars = start_map.map(function(name, index){
+    const stars = this.props.stars ? this.props.stars : 0;
```

Comment:
Thanks for cleaning up the horrible hack of how I process Stars. But right now the user can enter a value > 5 for stars.

We need to make sure that 5 is the max and 0 is the min:

```
    if (stars > 5) {
      stars = 5
    } else if (stars < 0) {
      stars = 0
    }
```